xbmc/filesystem/ZeroconfDirectory.cpp

```diff
@@ -48,8 +48,6 @@ namespace
       return "SAMBA";
     else if(fcr_service_type == "_ftp._tcp.")
       return "FTP";
-    else if(fcr_service_type == "_htsp._tcp.")
-      return "HTS";
```

Comment:
@opdenkamp Just realised that this could have killed your avahi discovery feature for pvr.hts?